docs/topics/development/services.rst

```diff
@@ -1,12 +0,0 @@
-.. _services:
-
-==========================
-Services
-==========================
-
-The services directory contain a special separate piece of code that deals with the update service Firefox calls to get updates about installed add-ons (though the ``extensions.update.background.url`` and ``extensions.update.background.url`` preferences).
-
-In dev/stage/prod, this would have its own domain, but locally, we re-use the same as
-the rest of addons-server, and our nginx configuration answers using the separate ``versioncheck`` wsgi service for all requests with a /update/ prefix as their path. This minimal configuration has autoreload disabled to stay as lean as possible, so to you'll need to manually restart the web docker service to see changes.
-
-A typical URL to test with would look like this: ``http://olympia.test/update/?reqVersion=1&id=addon@guid&version=0.1&appID={ec8030f7-c20a-464f-9b0e-13a3a9e97384}&appVersion=99.0``
```

Comment:
Does any of this need to go elsewhere?  We still have an update service, just not written as a separate bunch of non-django python files.

Reply:
Now it has a proper view/url that can be resolved/reversed so that doc is less useful... Maybe I should add an example of the query parameters in the view itself.

Reply:
https://github.com/mozilla/addons-server/pull/20086/commits/2bd62966c775efd0c3b6dfe0039b05b04e8f48da